docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+:ref-prefix:
+    habitat_sim.sim
+    habitat_sim.attributes
+
+Using JSON Files to configure Attributes
+########################################
+
+:summary: Attributes templates are used in Habitat to configure the simulation world and specify characteristics for scenes and objects. This document describes the appropriate tags to be used when authoring JSON files to properly customize these various Attributes templates.
+
+.. contents::
+    :class: m-block m-default
+
+Attributes templates provide a mechanism by which the various constructions in Habitat can be constructed with user-specified characteristics, either at program start or on the fly.
+
+`Physics Manager Attributes`_
+=============================
+Physics Manager Attributes templates describe quantities pertinent to building the simulation world.  The descriptor JSON files used for these attributes should be named in the following format : 
+
+ 	<worldname>.phys_scene_config.json
+
+
+Below are the supported JSON tags for Physics Manager Attributes templates, and their meanings.
+
+"physics simulator"
+	- string
+	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+	
+"gravity"  
+	- 3-vector
+	- Gravity to use in scene for physical modeling.
+
+"timestep" 
+	- double
+	- The timestep to use for forward simulation.
+ 
+"friction coefficient"
```

Comment:
I'm surprised to see this at the physics-world level. Is this just a default in case a physics object doesn't specify it?